### PR TITLE
asset tables

### PR DESCRIPTION
**Pull request by benauriemma**

### CODE REVIEW

File: rackcity/api/serializers/asset_serializers.py [hunk 1]
```diff
@@ -150,6 +150,7 @@ class Meta:
             "rack_position",
             "chassis",
             "chassis_slot",
+            "datacenter",
```
Comment (benauriemma) [inline rackcity/api/serializers/asset_serializers.py:153]: for column order

File: rackcity/migrations/0051_auto_20200415_0253.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            model_name='decommissionedasset',
+            name='datacenter',
+            field=django.contrib.postgres.fields.jsonb.JSONField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='decommissionedasset',
+            name='offline_storage_site',
+            field=django.contrib.postgres.fields.jsonb.JSONField(blank=True, null=True),
+        ),
+    ]
```
Comment (benauriemma) [inline rackcity/migrations/0051_auto_20200415_0253.py:24]: Database freeze? Never heard of her

File: rackcity/utils/query_utils.py [hunk 3]
```diff
@@ -216,6 +222,21 @@ def get_datacenter_filter_arguments(match_type, value):
     return datacenter_filter_args
 
 
+def get_offline_site_filter_arguments(match_type, value):
```
Comment (juliasaveliff) [inline rackcity/utils/query_utils.py:225]: noice

File: src/components/elementView/detailedView/assetView/assetView.tsx [hunk 4]
```diff
@@ -519,8 +531,15 @@ export class AssetView extends React.PureComponent<
...
         params = this.props.match.params;
         this.updateAssetData(params.rid);
+        this.props.markTablesStale([
+          TableType.RACKED_ASSETS,
+          TableType.STORED_ASSETS,
+          TableType.DECOMMISSIONED_ASSETS,
+        ]);
       })
       .catch((err) => {
+        console.log("there was an error: ");
```
Comment (juliasaveliff) [inline src/components/elementView/detailedView/assetView/assetView.tsx:541]: console.logs!

File: src/components/elementView/detailedView/assetView/assetView.tsx [hunk 5]
```diff
@@ -538,4 +557,13 @@ const mapStatetoProps = (state: any) => {
   };
 };
 
-export default withRouter(connect(mapStatetoProps)(AssetView));
+const mapDispatchToProps = (dispatch: any) => {
+  return {
+    markTablesStale: (staleTables: TableType[]) =>
+      dispatch(actions.markTablesStale(staleTables)),
```
Comment (juliasaveliff) [inline src/components/elementView/detailedView/assetView/assetView.tsx:563]: hell ya full stack benny

File: src/components/elementView/elementTable.tsx [hunk 6]
```diff
@@ -518,7 +527,18 @@ class ElementTable extends React.Component<
   };
 
   componentDidUpdate() {
-    if (this.props.shouldUpdateData && !this.props.data) {
+    let dataIsStale =
```
Comment (juliasaveliff) [inline src/components/elementView/elementTable.tsx:530]: I can hear pitbull saying stale like dale

**Pull request merged**